modules/statics/src/coins.ts

```diff
@@ -1006,6 +1006,26 @@ export const coins = CoinMap.fromCoins([
     BaseUnit.TCOREUM,
     COSMOS_SIDECHAIN_FEATURES
   ),
+  account(
+    '9fa0f191-4eed-4030-864a-d14bbd98c8af',
+    'rune-native',
+    'Rune',
+    Networks.main.rune,
+    8,
+    UnderlyingAsset.RUNE,
+    BaseUnit.RUNE,
+    COSMOS_SIDECHAIN_FEATURES
```

Comment:
i dont think this is a cosmos like coin.

Reply:
It is a cosmos like chain, confirmed with product team also with the foundation.